### ISSUE TITLE
Rename “Total Alt Tag Errors” column

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#536]
... to "Total Missing Image Descriptions".  

On the [Agency tab](https://staging.pulse.cio.gov/accessibility/agencies/) of the accessibility section

**Comment by kengdoj** [comment]
Hi, I don't have the full specs on what Pulse is checking regarding image description techniques. If it is only checking for ALT, I would recommend leaving the error as described. If it can check for title, ARIA, etc. then changing the description as suggested would be ok.

**Comment by gbinal** [comment]
Totally fair point, @kengdoj.  The team thinks that this will be a more appropriate column header, but I'm making a note in #503 to update the about page to make clear what is being scanned here.

**Comment by gbinal** [comment]
This is addressed by #549

**Comment by gbinal** [comment]
done

**Comment by gbinal** [comment]
This remains an issue in the expanded views for domains and on the individual error pages.

**Comment by gbinal** [comment]
Done

**Comment by gbinal** [comment]
Came back into play with the last data update

**Comment by gbinal** [comment]
This is addressed.

**Comment by gbinal** [comment]
I've been manually editing this each time but we should integrate the change into the scripts and scans so that there's no need to do manually anymore.

**Comment by gbinal** [comment]
This is now fully addressed and can be closed.
